src/main/java/uk/ac/ebi/atlas/home/HomeController.java

```diff
@@ -1,19 +1,29 @@
 package uk.ac.ebi.atlas.home;
 
+import com.google.gson.JsonParser;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 import uk.ac.ebi.atlas.controllers.HtmlExceptionHandlingController;
 import uk.ac.ebi.atlas.experiments.ExperimentInfoListService;
 import uk.ac.ebi.atlas.utils.ExperimentInfo;
 
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+
 import static uk.ac.ebi.atlas.home.AtlasInformationDataType.EFO;
 import static uk.ac.ebi.atlas.home.AtlasInformationDataType.EG;
 import static uk.ac.ebi.atlas.home.AtlasInformationDataType.ENSEMBL;
 import static uk.ac.ebi.atlas.home.AtlasInformationDataType.WBPS;
 
 @Controller
 public class HomeController extends HtmlExceptionHandlingController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(HomeController.class);
+    private static final String URL = "http://ftp.ebi.ac.uk/pub/databases/microarray/data/atlas/sc_experiments/cell_stats.json";
```

Comment:
The path shouldn’t be absolute, but relative to the property `data.file.locations` as defined in the `configuration.properties` file.